### ISSUE TITLE
Linux block devices now count flush requests, breaking "diskstats" plugin

### ISSUE THREAD

**Issue by Hexhu** [munin-monitoring/munin#1300]
**Describe the bug**

`/etc/munin/plugins/diskstats` will report `"'/sys/block/<dev>/stat' contains the wrong amount of values. Aborting at /etc/munin/plugins/diskstats line 432.`" after linux 5.5. This disables the `diskstats` plugin.

The documentation of the two newly added fields, `flush I/Os` and `flush ticks` can be found at https://github.com/torvalds/linux/blob/master/Documentation/block/stat.rst.

This is a part of a larger commit "block: add iostat counters for flush requests":
https://github.com/torvalds/linux/commit/b6866318657717c8914673a6394894d12bc9ff5e
The ABI of `procfs` and `sysfs` have been modified to include this very useful feature.

**To Reproduce**
1. `ln -s /usr/lib/munin/plugins/diskstats /etc/munin/plugins/diskstats`
2. `systemctl restart munin-node.service`
3. `tail -f /var/log/munin/munin-node.log`

**Expected behavior**
No error should be reported :3

**Screenshots & Logs**
During setup, the `autoconf` tool will complain:
```
# munin-node-configure --shell
# The following plugins caused errors:
...
# diskstats:
# 	Non-zero exit during autoconf (1)
```
After manually enabling `diskstats` and doing a `tail -f /var/log/munin/munin-node.log`, the error will be like:
```
# tail -f /var/log/munin/munin-node.log
2020/03/16-00:09:06 [1133148] Error output from diskstats:
2020/03/16-00:09:06 [1133148] 	'/sys/block/md127/stat' contains the wrong amount of values. Aborting at /etc/munin/plugins/diskstats line 432.
2020/03/16-00:09:06 [1133148] 		main::read_sysfs() called at /etc/munin/plugins/diskstats line 458
2020/03/16-00:09:06 [1133148] 		main::parse_diskstats() called at /etc/munin/plugins/diskstats line 530
2020/03/16-00:09:06 [1133148] 		main::fetch_device_counters() called at /etc/munin/plugins/diskstats line 35
2020/03/16-00:09:06 [1133148] Service 'diskstats' exited with status 25/0.
```

**Desktop:**
 - OS & Distribution Version: Linux 5.5.6, on Arch Linux
 - Munin Version: 2.0.52

**Comment by Hexhu** [comment]
I will be working this issue by following https://github.com/munin-monitoring/munin/commit/a913c523976bb13b2a34968ee5fe3b55d2b0918f
I have little experience in perl, but I will give it a try anyways, because `diskstats` is sooo useful.

**Comment by Hexhu** [comment]
Seems to be already addressed by https://github.com/munin-monitoring/munin/issues/1275 and solved by https://github.com/munin-monitoring/munin/commit/c3e46710fd7b08385ff81995208e65ba423dd2d5 on branch stable-2.0.
Although the moench-tegeder's PR https://github.com/munin-monitoring/munin/pull/1277 is still pending for master to be ready for linux 5.5.
I am closing this issue and my PR.

**Comment by sumpfralle** [comment]
Thank you for starting to work on the issue - and sorry for the confusion due to the non-merged changes from `stable-2.0`.

Now I merged the two outstanding releases from the `stable-2.0` branch (containing the relevant change). Sorry for the delay!
